sample_application/app.py

```diff
@@ -8,6 +8,7 @@
 from views import UnixTime, PrintArg, ExampleApiUsage
 from flask.ext.restful import Api
 from flask.ext.discoverer import Discoverer
+from database import db
```

Comment:
please rename `database.py` to `models.py`, since this is the standard naming convention in the flask/django MVC world.